.circleci/config.yml

```diff
@@ -0,0 +1,295 @@
+version: 2.1
+
+
+orbs:
+  mim:
+    commands:
+      install_erlang:
+        parameters:
+          otp:
+            type: string
+            description: Version of the Erlang package to install
+        steps:
+        - run:
+            name: Install Erlang
+            command: |
+              wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
+              sudo dpkg -i erlang-solutions_1.0_all.deb
+              sudo apt-get update
+              sudo apt-get install -y esl-erlang=<<parameters.otp>>
+      fetch_packages:
+        steps:
+        - run:
+            name: Install basic packages
+            command: |
+              sudo apt install unixodbc-dev
```

Comment:
Let's use `apt-get` instead as the `apt` is targeted to be used by humans, and thus officially not having stable CLI API (in contrast to `apt-get`).
Also, let's add `-y` flag to make sure this doesn't fail when dependencies change for this package.